lambda-ci-resource.yaml

```diff
@@ -0,0 +1,32 @@
+include:
+  - https://raw.githubusercontent.com/Cimpress-MCP/gitlab-continuous-integration/master/base-ci-resource.yaml
+
+publish:
+  stage: publish
+  script:
+    - echo "No-op publish for lambda deploy"
+
+deploy-int:
+  extends: .deploy

```

Comment:
Not sure you can use yaml inheritance unless the thing you are inheriting from is in this file.

Reply:
This isn't YAML inheritance, though... it's GitLab's special inheritance.

Reply:
Ah, I see

https://docs.gitlab.com/ee/ci/yaml/#using-extends-and-include-together

Thanks!